qa/tasks/ceph_test_case.py

```diff
@@ -148,6 +148,10 @@ def _init_mon_manager(self):
             from tasks.ceph_manager import CephManager
```

Comment:
Reviewing the older part of `_init_mon_manager` I see that you weren't the first to use `if 'Local' in str(type(self.ceph_cluster)):`. It doesn't make the code valid, but it explains the origin.

I would have this fixed. The manager instance should either be taken from the context, as I did originally, or it should implement the same class interface as `CephManager`, and should be injected via the module class variable, as I did with the other vstart_runner overrides.

No code outside of vstart_runner should be using classes defined there